xmake/plugins/project/vstudio/impl/vs201x.lua

```diff
@@ -192,19 +199,19 @@ function _make_custom_commands_for_objectrules(commands, target, sourcebatch, su
 end
 
 -- make custom commands
-function _make_custom_commands(target)
+function _make_custom_commands(target, vcxprojdir)
     local commands = {}
-    _make_custom_commands_for_target(commands, target, "before")
-    _make_custom_commands_for_target(commands, target)
+    _make_custom_commands_for_target(commands, target, "before", vcxprojdir)
+    _make_custom_commands_for_target(commands, target, nil, vcxprojdir)
```

Comment:
you can adjust vcxprojdir position to avoid nil argument.

```
_make_custom_commands_for_target(commands, target, vcxprojdir)
    _make_custom_commands_for_target(commands, target, vcxprojdir, "before")

```